extensions/arc/src/ui/dialogs/connectControllerDialog.ts

```diff
@@ -271,7 +272,15 @@ export class PasswordToControllerDialog extends ControllerDialogBase {
 		}
 		const azdataApi = <azdataExt.IExtension>vscode.extensions.getExtension(azdataExt.extension.name)?.exports;
 		try {
-			await azdataApi.azdata.login(this.urlInputBox.value!, this.usernameInputBox.value!, this.passwordInputBox.value);
+			await azdataApi.azdata.login(
+				this.urlInputBox.value!,
+				this.usernameInputBox.value!,
+				this.passwordInputBox.value,
+				{
+					'KUBECONFIG': this.kubeConfigInputBox.value!,
+					'KUBECTL_CONTEXT': this.clusterContextRadioGroup.value!
+				}
+			);
 		} catch (e) {
 			if (getErrorMessage(e).match(/Wrong username or password/i)) {
```

Comment:
Probably should change the default error message displayed - "Error encountered while verifying password" isn't really going to be necessarily correct. Just something generic like "Error logging into controller" is probably fine

Reply:
fixed